exporting/read_config.c

```diff
@@ -432,7 +439,14 @@ struct engine *read_exporting_config()
         tmp_instance->config.prefix = strdupz(exporter_get(instance_name, "prefix", "netdata"));
 
 #ifdef ENABLE_HTTPS
-        if (tmp_instance->config.type == EXPORTING_CONNECTOR_TYPE_OPENTSDB_USING_HTTP && !strncmp(tmp_ci_list->local_ci.connector_name, "opentsdb:https", 14)) {
+        if ((tmp_instance->config.type == EXPORTING_CONNECTOR_TYPE_GRAPHITE_HTTP &&
```

Comment:
@ilyam8 take a look here! :smile:

Reply:
at least it is easy to follow 😉 

```c
 (a && b) ||
 (c && d) ||
 (e && f) ||
 ...
```

Reply:
I'm always afraid of this type of magic numbers. Why not `strlen()` instead?

Reply:
Fixed